src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -94,6 +102,87 @@ class TransactionDetailsComponent extends React.Component<Props, State> {
     return await wallet.accelerate(transaction)
   }
 
+  /**
+   * Given an EdgeTxAction, returns the display value for pre-filling the 'Category' tile
+   */
+  getTxActionSplitCategory = (action: EdgeTxAction | undefined): SplitCategory | undefined => {
```

Comment:
Can we fold this logic into the `splitCategory` function, or at least move it into the same file so it's accessible as a utility function?

Maybe we could call it `getTransactionCategory(walllet: EdgeCurrencyWallet, tx: EdgeTransaction): SplitCategory`, and then this function could look at both `tx.action` and `tx.metadata.category`, which would both be available in that case.

That way, we can utilize this function in the CSV export / QBO export functions, or any other places where we need the category.